DEPENDENCIES.txt

```diff
@@ -32,6 +32,7 @@ FindBin
 Getopt::Long
 IO::Socket::INET6
 IO::Socket::SSL
+IPC::Run::Debug
```

Comment:
Ah, didn't notice that. Anyway I think, requiring ::Debug variant and counting on that it will pull the base is not quite correct. (And on openSUSE IPC::Run::* is provided by the same package)